server.js

```diff
@@ -4,8 +4,8 @@ const shortid = require('shortid');
 app.use(express.json());
 
 app.locals.notes = [
-  { id: shortid.generate(), title: 'fakeTitle', body: [{id: shortid.generate(), text: 'faketext'}] },
-  { id: shortid.generate(), title: 'fakeTitle2', body: [{id: shortid.generate(), text: 'faketext2'}] }
+  { id: '1', title: 'fakeTitle', body: [{id: shortid.generate(), text: 'faketext'}] },
```

Comment:
Thank you for testing out the DELETE request with the ids as the strings '1' and '2'. We can change these back once we finish testing each response.